package.json

```diff
@@ -121,6 +121,7 @@
     "collectCoverage": true
   },
   "dependencies": {
-    "postcss-less": "^6.0.0"
+    "postcss-less": "^6.0.0",
+    "swagger-ui-dist": "^5.9.1"
```

Comment:
This should be installed as a dev-dependency not a dependency.
You can fix by uninstalling and reinstalling with `npm install [package_name] --save-dev`

Reply:
okay.
I will do this.